styles/learn-x-in-y-minutes/catppuccin.user.css

```diff
@@ -0,0 +1,216 @@
+/* ==UserStyle==
+@name Learn X in Y Minutes Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/learn-x-in-y-minutes
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/learn-x-in-y-minutes
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/learn-x-in-y-minutes/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Alearn-x-in-y-minutes
+@description Soothing pastel theme for Learn X in Y Minutes
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document domain("learnxinyminutes.com") {
+  @import url("https://python.catppuccin.com/pygments/catppuccin-variables.important.css");
+
+  html.light {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+
+  html.dark {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+
+  html:not(.dark, .light) {
+    @media (prefers-color-scheme: light) {
+      #catppuccin(@lightFlavor, @accentColor);
+    }
+    @media (prefers-color-scheme: dark) {
+      #catppuccin(@darkFlavor, @accentColor);
+    }
+  }
+
+  #catppuccin(@lookup, @accent) {
+    @rosewater: @catppuccin[ @@lookup][ @rosewater];
+    @flamingo: @catppuccin[ @@lookup][ @flamingo];
+    @pink: @catppuccin[ @@lookup][ @pink];
+    @mauve: @catppuccin[ @@lookup][ @mauve];
+    @red: @catppuccin[ @@lookup][ @red];
+    @maroon: @catppuccin[ @@lookup][ @maroon];
+    @peach: @catppuccin[ @@lookup][ @peach];
+    @yellow: @catppuccin[ @@lookup][ @yellow];
+    @green: @catppuccin[ @@lookup][ @green];
+    @teal: @catppuccin[ @@lookup][ @teal];
+    @sky: @catppuccin[ @@lookup][ @sky];
+    @sapphire: @catppuccin[ @@lookup][ @sapphire];
+    @blue: @catppuccin[ @@lookup][ @blue];
+    @lavender: @catppuccin[ @@lookup][ @lavender];
+    @text: @catppuccin[ @@lookup][ @text];
+    @subtext1: @catppuccin[ @@lookup][ @subtext1];
+    @subtext0: @catppuccin[ @@lookup][ @subtext0];
+    @overlay2: @catppuccin[ @@lookup][ @overlay2];
+    @overlay1: @catppuccin[ @@lookup][ @overlay1];
+    @overlay0: @catppuccin[ @@lookup][ @overlay0];
+    @surface2: @catppuccin[ @@lookup][ @surface2];
+    @surface1: @catppuccin[ @@lookup][ @surface1];
+    @surface0: @catppuccin[ @@lookup][ @surface0];
+    @base: @catppuccin[ @@lookup][ @base];
+    @mantle: @catppuccin[ @@lookup][ @mantle];
+    @crust: @catppuccin[ @@lookup][ @crust];
+    @accent-color: @catppuccin[ @@lookup][ @@accent];
```

Comment:
What in the world is going on with the formatting here 😅 can you format this file with Prettier or otherwise fix this?